dts/bindings/ethernet/atmel,sam-gmac.yaml

```diff
@@ -15,3 +16,32 @@ properties:
       type: int
       required: true
       description: Number of hardware TX and RX queues
+
+    local-mac-address:
+      type: uint8-array
+      description:
+        Specifies the MAC address that was assigned to the network device
+
+    max-frame-size:
+        type: int
+        description:
+          Maximum transfer unit (IEEE defined MTU), rather than the
+          maximum frame size (there\'s contradiction in the Devicetree
+          Specification). The current supported values are 1518, 1536
+          and 10240 (jumbo frames).
+        default: 1518
+
+    max-speed:
+        type: int
+        description:
+          Specifies maximum speed in Mbit/s supported by the device.
+        default: 100
+
+    phy-connection-type:
+        type: string
+        description:
+          Operation mode of the PHY interface
+        enum:
+          - rmii
+          - mii
```

Comment:
Shouldn't this be:
```
- "rmii"
- "mii"
```
(cc @galak @ulfalizer)

Reply:
fixed!